packages/lab/src/__tests__/__e2e__/date-picker/DatePicker.cy.tsx

```diff
@@ -0,0 +1,205 @@
+import { composeStories } from "@storybook/react";
+import { ChangeEvent } from "react";
+import * as datePickerStories from "@stories/date-picker/date-picker.stories";
+import { checkAccessibility } from "../../../../../../cypress/tests/checkAccessibility";
+import {
+  CalendarDate,
+  DateFormatter,
+  DateValue,
+  getLocalTimeZone,
+  startOfMonth,
+  today,
+} from "@internationalized/date";
+
+const composedStories = composeStories(datePickerStories);
+const { Default, Range } = composedStories;
+
+const testDate = new CalendarDate(2000, 2, 1);
+const testInput = "02-feb-2000";
+const rangeTestInput = "02-mar-2000";
+const localTimeZone = getLocalTimeZone();
+const currentLocale = navigator.languages[0];
+const formatDate = (date: DateValue, options?: Intl.DateTimeFormatOptions) => {
+  const formatter = new DateFormatter(currentLocale, options);
+  return formatter.format(date.toDate(localTimeZone));
+};
+const formatInput = (date: DateValue): string =>
+  new DateFormatter("EN-GB", {
+    day: "2-digit",
+    month: "short",
+    year: "numeric",
+  }).format(date.toDate(localTimeZone));
+
+const formatDay = (date: DateValue) => {
+  return formatDate(date, {
+    day: "2-digit",
+    month: "long",
+    year: "numeric",
+  });
+};
+
+describe("GIVEN a DatePicker", () => {
+  checkAccessibility(composedStories);
+
+  describe("WHEN single datepicker is mounted", () => {
+    it("THEN it should mount with the specified defaultStartDate", () => {
+      cy.mount(<Default defaultStartDate={testDate} />);
```

Comment:
`defaultStartDate` feels uncomfortable to me as a single date picker's api. Is there a need to split 2 components?

Reply:
makes sense, it probably should follow calendar usage of selectedDate, noted